internal/daemon/controller/controller.go

```diff
@@ -264,6 +264,7 @@ func New(ctx context.Context, conf *Config) (*Controller, error) {
 		return nil, fmt.Errorf("error initializing rate limiter: %w", err)
 	}
 	c.rateLimiter.Store(rateLimiter)
+	ratelimit.WriteLimitsSysEvent(c.baseContext, rateLimits, conf.RawConfig.Controller.ApiRateLimiterMaxEntries)
```

Comment:
`ratelimit.WriteLimitsSysEvent` returns an error. Can we handle the error? If not, can we `_ = ratelimit.WriteLimitsSysEvent` and add a comment to explain?